pkg/service/wire.go

```diff
@@ -9,14 +9,14 @@ import (
 	"github.com/livekit/livekit-server/pkg/routing"
 )
 
-func InitializeServer(conf *config.Config, currentNode routing.LocalNode) (*LivekitServer, error) {
+func InitializeServer(conf *config.Config, currentNode routing.LocalNode, isTest bool) (*LivekitServer, error) {
```

Comment:
this is changed InitializeServer func

Reply:
we just removed the StaticProvider code to simplify things. sorry, this ended up being a significant change. could you merge and see if issues remain?

Reply:
no problem. I think remove StaticProvider version is better than before.
Deleting the StaticProvider and the CreateKeyProvider are included in the wire, so it's easier to work with.